### ISSUE TITLE
Main Website; WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by A11Ybot-OP** [istio/community#783]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.panel-btn.btn)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Istio - https://istio.io/

Element path: .panel--dark.panel[href$="service-mesh"] > .panel-body > .panel-btn.btn

Snippet: `<span class="btn panel-btn">Learn more</span>`

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.19 (foreground color: #7dc5f2, background color: #3d5080, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 104.0.0.0

**Comment by A11Ybot-OP** [comment]
5 other similar issue on the website
